test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -18,6 +18,37 @@ class BulkRepublisherTest < ActiveSupport::TestCase
 
       BulkRepublisher.new.republish_all_documents
     end
+
+    test "doesn't queue non-editionable content" do
+      BulkRepublisher.any_instance.stubs(:non_editionable_content_types).returns(%w[Contact Role])
+
+      Whitehall::PublishingApi.expects(:bulk_republish_async).with(create(:contact)).never
+      Whitehall::PublishingApi.expects(:bulk_republish_async).with(create(:role)).never
+
+      Contact.any_instance.stubs(:can_publish_to_publishing_api?).returns(true)
+      Role.any_instance.stubs(:can_publish_to_publishing_api?).returns(true)
+
+      BulkRepublisher.new.republish_all_documents
```

Comment:
Mind adding a quick line to the commit message to explain what `republish_all_documents` is and why it shouldn't include non-editionable docs or individual pages please :)